samples/sensor/hs300x/README.rst

```diff
@@ -0,0 +1,65 @@
+.. _renesas_hs300x_sample:
+
+RENESAS_HS300X Sample
+#####################
```

Comment:
Nit: typically only sensor model is used for naming sensor samples 

```suggestion
HS300X Sample
#############
```

Reply:
done